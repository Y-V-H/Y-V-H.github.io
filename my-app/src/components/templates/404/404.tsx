import React from "react";
import { Header } from '../../atoms/header/header'

export const PageNotFound = () => {
    return (
        <>
            <Header size='h1' children="page not found"/>
        </>
    )
}