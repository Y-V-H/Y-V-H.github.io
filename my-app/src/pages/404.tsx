import { use, useEffect } from 'react';
import { Header } from '../components/atoms/header/header';
import { useRouter } from 'next/router';
import { SearchBlock } from '../components/organisms/search-block/search-block';

export default function PageNotFound() {
  const router = useRouter();

  useEffect(() => {
    setTimeout(()=>{
      router.push('/')
    },3000)
  },[router]);

  return (
    <div className='container'>
      <Header size="h1">Page not found</Header>
    </div>
  )
}