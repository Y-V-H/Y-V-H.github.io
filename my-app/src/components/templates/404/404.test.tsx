import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom'
import { PageNotFound } from './404';

test('should render PageNotFound template', () => {
    const component = renderer.create(
        <PageNotFound />
    );
    expect(component.toJSON()).toMatchSnapshot();
});