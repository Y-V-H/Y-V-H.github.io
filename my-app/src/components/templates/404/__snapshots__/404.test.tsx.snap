// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFound template 1`] = `
<h1
  className="a-header a-header--h1 undefined"
>
  page not found
</h1>
`;
